atomistics/calculators/sphinxdft.py

```
@@ -0,0 +1,136 @@
+
+from atomistics.calculators.interface import get_quantities_from_tasks
+from atomistics.calculators.wrapper import as_task_dict_evaluator
+from atomistics.shared.output import OutputStatic
+
+
+def _write_input(
+    structure: Atoms,
+    working_directory: str,
+    maxSteps: int = 100,
+    eCut: float = 25,
+    kpoint_coords: list[float, float, float] = [0.5, 0.5, 0.5],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)